sygnal/apnspushkin.py

```diff
@@ -239,10 +248,7 @@ async def _dispatch_request(
         else:
             # .description corresponds to the 'reason' response field
             span.set_tag("apns_reason", response.description)
-            if (
-                code == self.TOKEN_ERROR_CODE
-                or response.description == self.TOKEN_ERROR_REASON
-            ):
+            if (code, response.description) in self.TOKEN_ERRORS:
```

Comment:
There's an existing test that covers the `(410, "Unregistered")` behaviour.

Reply:
Previously we would error on `410` or `Unregistered`, but from their docs these should always be grouped so this shold be OK. 👍 

Is it worth logging or anything that we're deactivating this?

Reply:
Good spot, I actually misread the condition as an `and`, since I wasn't expecting an `or`!

Some digging reveals that we're already logging the tokens we reject here:
https://github.com/matrix-org/sygnal/blob/main/sygnal/http.py#L283-L287

Reply:
That log-line is really confusing it says it was successful, but there were rejections.

I guess my question is -- how is someone supposed to debug *why* their request was rejected? Or does that not matter?